Fix:

error: conflicting types for 'ohcount_hash_parser_from_language'

https://github.com/blackducksoftware/ohcount/commit/e5f6d25a9cd7c721fea67f35d371405e87fc8144
--- src/languages.h.orig
+++ src/languages.h
@@ -126,6 +126,6 @@
 
 // For gperf.
 struct LanguageMap { const char *key; const char *name; const char *nice_name; int category; };
-struct LanguageMap *ohcount_hash_language_from_name(register const char *str, register unsigned int len);
+struct LanguageMap *ohcount_hash_language_from_name(register const char *str, register size_t len);
 
 #endif
--- src/parser.c.orig
+++ src/parser.c
@@ -7,7 +7,7 @@
 #include "log.h"
 #include "hash/parser_hash.h"
 
-struct ParserMap * ohcount_hash_parser_from_language (register const char *str, register unsigned int len);
+struct ParserMap * ohcount_hash_parser_from_language (register const char *str, register size_t len);
 
 int ohcount_parse(SourceFile *sourcefile, int count,
                   void (*callback) (const char *, const char *, int, int,
